packages/picongpu/picongpu.profile

```diff
@@ -7,3 +7,11 @@ spack load @PIC_SPACK_COMPILER@ >/dev/null 2>&1 || echo "using default compiler"
 
 # required: PIConGPU and its dependencies
 spack load -r @PIC_SPACK_SPEC@
+
+# activate bash completion if available
+BASH_COMP_FILE=$PICSRC/bin/picongpu-completion.bash
+if [ -f $BASH_COMP_FILE ] ; then
+    source $BASH_COMP_FILE
+else
+    echo "bash completion file '$PICSRC/bin/picongpu-completion.bash' not found." >&2
```

Comment:
If you have some more minutes this line can also be `echo "bash completion file '$BASH_COMP_FILE' not found." >&2`

Reply:
I thought that this would mean, that the user sees the variable in the output and not the full path, so I just skipped it here. But I'm wrong and it should echo the full path. So I'll do it in a second :)